suprb/optimizer/rule/generation_operator/__init__.py

```diff
@@ -0,0 +1 @@
+from .base import GenerationOperator
```

Comment:
I don't think this needs to be a separate module. For now it can just live within optimizer/rule

Reply:
Readibility really plummets if we have this as a module, imho